src/Item/Item.service.ts

```diff
@@ -546,17 +537,9 @@ export class ItemService {
             decodedURN.item_urn_suffix
           )
 
-          // Check if the item's URN in the DB is published before changing it
-          if (await thirdPartyAPI.itemExists(dbItemURN)) {
-            throw new ThirdPartyItemAlreadyPublishedError(
-              dbItem.id,
-              dbItemURN,
-              ItemAction.UPSERT
-            )
-          }
-
           // Check if the new URN is not already in use
-          if (await thirdPartyAPI.itemExists(itemURN)) {
+          const [wearable] = await peerAPI.fetchWearables([itemURN])
```

Comment:
In theory, we could do the same check by checking if the `urn_suffix` exists for any item on the given collection.

Reply:
That's true. I wasn't 100% sure it was reliable. The database *should* contain all `urn_suffix`es but the ultimate source of truth is the Catalyst.

You think it'd be safe to change it?

Reply:
It is a good question really, I'm not 100% sure about it.